src/QCToolsBundle/Resources/views/Download/macTable.html.twig

```diff
@@ -10,9 +10,14 @@
     <td colspan="5">&nbsp;<br>&nbsp;</td>
 </tr>
 <tr>
-    <th rowspan="2"><br>10.10&nbsp;(Yosemite),<br>10.11&nbsp;(El&nbsp;Captain),<br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
-    <th rowspan="2">x86_64</th>
-    <td><abbr title="Graphical User Interface">GUI</abbr></td>
+    <th rowspan="3"><br>10.10&nbsp;(Yosemite),<br>10.11&nbsp;(El&nbsp;Captain),<br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
+    <th rowspan="3">x86_64</th>
+    <td><abbr title="Graphical User Interface">GUI</abbr> (mac0S 10.12 and newer) </td>
+    <td><a href="//mediaarea.net/download/binary/qctools/1.1/QCTools_1.1_mac_1012.dmg">v1.1</a></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <td><abbr title="Graphical User Interface">GUI</abbr> (mac0S 10.10 and 10.11) </td>
```

Comment:
As it works with newer versions, I would list all of them (10.10 to 10.15).

Reply:
I have reworked my PR